try:
    a = int(input("Enter the no.:-"))
    print(a)

# Exception CRASH HOU DET NAHI
except Exception as a:
    print(a, "TRY AGAIN, This is wrong ")

    print("Thank you")


# #######                 NEW

try:
    a = int(input("this is the value:-"))
    print(a)

except Exception as a:
    print(a)

print("Thhs jdriipeyhid yhi ")
